src/rules/and/marginals.jl

```diff
@@ -0,0 +1,8 @@
+@marginalrule AND(:in1_in2) (
+    m_out::Bernoulli,
+    m_in1::Bernoulli,
+    m_in2::Bernoulli
+) = begin
+    pin1, pin2, pout = mean(m_in1), mean(m_in2), mean(m_out)
+    return Contingency([(1-pin1)*(1-pin2)*(1-pout) (1-pin1)*pin2*(1-pout); pin1*(1-pin2)*(1-pout) pin1*pin2*pout])
```

Comment:
The `Contingency` object is not automatically normalized. I think we want to extend the `contingency.jl` to do so, otherwise users will get tables, which are unnormalized, whereas they expect them to be normalized.

Reply:
Good point. I agree we should extend `contingency.jl`. Although in this case, this marginal is needed only for FE computations.